Dockerfile

```diff
@@ -1,92 +1,107 @@
-FROM ghcr.io/foundry-rs/foundry:master as builder
+# NOTE: The `latest` version of foundry does not include arm64 as a build platform, 
```

Comment:
Can we re-evaluate the style for this file? Try to break the code up so that each comment is preceded by a newline and followed by several lines of code. I make an exception for `NOTE` and `HACK` comments FWIW.

Reply:
Totally! It is somewhat inconsistent with the rest of our files.

Some context on why I changed it... empty lines within a stage in a multi-stage build make it super difficult to quickly grok where one stage begins and another ends. 

I took a peek at some of docker's documentation and it looks like they feel similarly, albeit most of what I saw was trivial with little commentary.

Personally not against whitespace within a stage, so long as we have something more visually significant to break up stages....

Dockerfiles only have single-line comments, so maybe a header for each stage like: 

```dockerfile
######################################
# Stage Name
# 
# what the stage does....
######################################
```